### ISSUE TITLE
How can we use mask rcnn weights in keras?

### ISSUE THREAD

**Issue by hrithikpuri5** [ibm-aur-nlp/PubLayNet#14]
Thank you for sharing the dataset! I want to use the maskrcnn with keras. How can we use the config and weights for the same?

**Comment by zhxgj** [comment]
@hrithikpuri5  I do not think you can, since the pre-trained model uses Detectron and Caffe2 (pytorch) framework. If you want to use keras with tensor flow backend, you will need to train the model on PubLayNet.
